### ISSUE TITLE
v0.3.0 does expose the `onForegroundAction` option

### ISSUE THREAD

**Issue by simoami** [tristanhimmelman/HidingNavigationBar#47]
Hello,
I installed v0.3.0 and it does not have the `onForegroundAction` option. the source code doesn't have the line:

```
    //Options
    public var onForegroundAction = HidingNavigationForegroundAction.Default
```

Could it be the wrong version being pushed?

pod install log:

``` sh
$ pod install
Updating local specs repositories

CocoaPods 1.1.0.beta.1 is available.
To update use: `gem install cocoapods --pre`
[!] This is a test version we'd love you to try.

For more information see http://blog.cocoapods.org
and the CHANGELOG for this version http://git.io/BaH8pQ.

Analyzing dependencies
Downloading dependencies
...
Installing HidingNavigationBar (0.3.0)
...
Generating Pods project
Integrating client project
Sending stats
Pod installation complete! There are 31 dependencies from the Podfile and 36 total pods installed.

```

**Comment by simoami** [comment]
?

**Comment by tristanhimmelman** [comment]
That addition actually hasn't been released yet.

I will look into updating to swift 3 and releasing this as well as soon as I can

**Comment by simoami** [comment]
Thanks!

**Comment by tristanhimmelman** [comment]
Just released v1.0.0 with swift 3 support and the feature mentioned above

**Comment by simoami** [comment]
Nice! that was fast 👍
